src/decisionengine/framework/logicengine/tests/test_construction.py

```diff
@@ -12,29 +12,28 @@ def test_wrong_configuration():
     """LogicEngine construction requires rules and facts;
     if we don't supply them it is an error."""
     with pytest.raises(KeyError):
-        LogicEngine({}, "test")
+        LogicEngine({})
 
 def test_trivial_configuration():
     """Logic engine constructed with trivial rules and facts."""
-    cfg = {"rules": {}, "facts": {}}
-    channelname = "test"
-    le = LogicEngine(cfg, channelname)
+    cfg = {"rules": {}, "facts": {}, "channel_name": "test"}
+    le = LogicEngine(cfg)
     assert le.produces() == ["actions", "newfacts"]
     assert le.consumes() == []
 
 def test_configuration_with_fact_using_function():
     facts = {"f1": "len(vals) == 10"}
     rules = {"r1": {"expression": "f1", "actions": ["launch"]}}
-    channelname = "test"
-    le = LogicEngine({"rules": rules, "facts": facts}, channelname)
+    channel_name = "test"
+    le = LogicEngine({"rules": rules, "facts": facts, "channel_name": channel_name})
```

Comment:
In all these tests I'd avoid the variable channel_name since it is used only once in the dictionary definition (it appears in 4 functions):
```python
le = LogicEngine({"rules": rules, "facts": facts, "channel_name": "test"})
```